### PR TITLE
add azure vnet

### PR DESCRIPTION
**Pull request by felipesborges**
there are a lot to improvments to be done,  but i'm providing the files anyway.

### CODE REVIEW

**Comment by benhur-araujo** [comment]
I reviewed your module's PR. We'll need to change this after adjusting the module

File: infrastructure/azure/eastus/resource-group/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,27 @@
+locals {
+  az-eastus-constants = yamldecode(file("../az-eastus-constants.yaml"))
+  global-constants = yamldecode(file("../../../global-constants.yaml"))
+}
+
+inputs = {
+  location = local.az-eastus-constants.region
+  #resource_group_name = "rg-${local.global-constants.project}-${local.az-eastus-constants.environment}-001"
```
Comment (benhur-araujo) [inline infrastructure/azure/eastus/resource-group/terragrunt.hcl:8]:
```suggestion

```

File: infrastructure/azure/eastus/resource-group/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    project           = local.global-constants.global-tags.project
+    team              = local.global-constants.global-tags.team
+    team-email        = local.global-constants.global-tags.team-email
+  }
+}
+
+
+terraform {
+  source = "git::https://github.com/conquerproject/tf-modules.git/azure-resource-group?ref=azure-resource-group-v0.1.0" 
+#  source = "../../../../../tf-modules/azure-resource-group/"
```
Comment (benhur-araujo) [inline infrastructure/azure/eastus/resource-group/terragrunt.hcl:21]:
```suggestion

```

File: infrastructure/azure/eastus/vnet/terragrunt.hcl [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    project           = local.global-constants.global-tags.project
+    team              = local.global-constants.global-tags.team
+    team-email        = local.global-constants.global-tags.team-email
+  }
+}
+
+
+terraform {
+   source = "git::https://github.com/conquerproject/tf-modules.git//azure-vnet?ref=azure-vnet-v0.1.0" 
+#   source = "../../../../../tf-modules/azure-vnet/"
```
Comment (benhur-araujo) [inline infrastructure/azure/eastus/vnet/terragrunt.hcl:32]:
```suggestion

```

File: infrastructure/azure/eastus/vnet/terragrunt.hcl [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  subnets = {
+    subnet-default = {
+      name            = "Default"
+      address_prefix  = ["10.0.1.0/24"]
+    }
+    subnet-k8s = {
+      name            = "whatever, just testing"
+      address_prefix  = ["10.0.2.0/24"]
+    }
+  }
```
Comment (benhur-araujo) [inline infrastructure/azure/eastus/vnet/terragrunt.hcl:20]: Beautiful. If this is working, you can remove the testing "subnet-k8s" subnet

File: infrastructure/azure/dev/az-dev-constants.yaml [hunk 5]
```diff
@@ -1,6 +1,8 @@
 subscription_id: "907cde31-ecc8-45c6-adb3-3c86455c885b"
 region: "eastus"
 environment: "dev"
+default_dev_rg: "rg-conquerplatform-dev-001"
+default_dev_vnet: "vnet-conquerplatform-dev-001"
```
[the remark this answers is not in the document]
Comment (benhur-araujo) [reply infrastructure/azure/dev/az-dev-constants.yaml:5]: I didn't get that, what do you mean?

**Review by leohauschild — APPROVED** [review]

**Pull request merged**